features/shared/folder_does_not_exist_on_main_branch/conflict.feature

```diff
@@ -13,11 +13,11 @@ Feature: git sync: syncing inside a folder that doesn't exist on the main branch
       | other-feature   | local and remote | other feature commit       | file2            |                 |
     And I am on the "current-feature" branch
     And I have an uncommitted file
-    When I run `git sync --all` in the "new_folder" folder
 
 
   @finishes-with-non-empty-stash
-  Scenario: result
+  Scenario: git-sync
+    When I run `git sync --all` in the "new_folder" folder
```

Comment:
I was preparing to add tests for git-ship, hack, and extract to the same feature. Didn't get to them yet.